flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/OperatorChain.java

```diff
@@ -713,13 +774,15 @@ private <T> WatermarkGaugeExposingOutput<StreamRecord<T>> createOutputCollector(
      */
     private <IN, OUT> WatermarkGaugeExposingOutput<StreamRecord<IN>> createOperatorChain(
             StreamTask<OUT, ?> containingTask,
+            StreamConfig prevOperatorConfig,
             StreamConfig operatorConfig,
             Map<Integer, StreamConfig> chainedConfigs,
             ClassLoader userCodeClassloader,
             Map<IntermediateDataSetID, RecordWriterOutput<?>> recordWriterOutputs,
             List<StreamOperatorWrapper<?, ?>> allOperatorWrappers,
             OutputTag<IN> outputTag,
-            MailboxExecutorFactory mailboxExecutorFactory) {
+            MailboxExecutorFactory mailboxExecutorFactory,
+            boolean shouldAddMetric) {
```

Comment:
```suggestion
            boolean shouldAddMetricForPrevOperator) {
```

Reply:
Thanks for the suggestion. It is fixed now.